src/storage/src/store_impl.rs

```diff
@@ -392,7 +392,17 @@ pub mod verify {
         }
     }
 
-    impl<A: LocalStateStore, E: LocalStateStore> LocalStateStore for VerifyStateStore<A, E> {}
+    impl<A: LocalStateStore, E: LocalStateStore> LocalStateStore for VerifyStateStore<A, E> {
+        define_local_state_store_associated_type!();
+
```

Comment:
It may not be possible because the return value of `surely_not_have` is implementation specific and cannot be consistent across different state store backend :(